tests/framework/types.go

```diff
@@ -57,6 +57,8 @@ type OsmTestData struct {
 	CleanupKindCluster             bool   // Cleanup kind cluster upon test finish
 	ClusterVersion                 string // Kind cluster version, ex. v1.20.2
 
+	ClusterOS string // The operating system of the working nodes in the cluster. Mixed OS traffic is not supported.
```

Comment:
Please remove this flag from this PR and scope this PR to just adding linux node selectors to the tests.

Reply:
I see this is just part of the config, not exposed as a flag.